Formula/brainfuck.rb

```diff
@@ -0,0 +1,27 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
+  homepage "https://github.com/fabianishere/brainfuck"
+  url "https://github.com/fabianishere/brainfuck/archive/2.7.0.tar.gz"
+  sha256 "445e00ef848f8b9a3ac101872ac08046b32b3477520ff06804d0ede5c3654c63"
+  head "https://github.com/fabianishere/brainfuck.git"
+
+  option "with-debug", "Compile interpreter with debug support"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args
+    args << "-DENABLE_DEBUG=ON" if build.with? "debug"
+
+    system "cmake", ".", *args
+    system "make"
+    bin.install "brainfuck"
```

Comment:
No `make install` here?

Reply:
There is no upstream support (yet), although I have a CMake configuration for this project that has a `make install` option . I've seen a ton of other formula do this so I suppose it's fine though (and this method actually works too, so seems okay to me?).

Reply:
Yeah, I was mostly asking because `make install` is nicer for everyone involved, but as far as I know Homebrew hasn't made such a thing mandatory for new formulae, so.

Reply:
`make install` or `install.sh` are basically mandatory for new formulae.